tests/README.md

```diff
@@ -6,50 +6,46 @@ language.
 Tests are declared inside plain Catala files with the following format:
 
 ~~~markdown
-```catala-test {id="TEST-IDENT"}
-catala ARGS
+```catala-test-inline
+$ catala ARGS
+... output from the catala command ...
 ```
 ~~~
 
-The output of running catala with the given `ARGS` for a `foo.catala_en` file is
-expected to be in an `output/foo.catala_en.TEST-IDENT` file (use
-`CLERK_OPTS=--reset` to (re-)generate it).
-
-You can use `CATALA_OPTS="..." make ...` to pass in Catala compiler options when
-debugging.
-
 ## Workflow for adding new tests
 
 1. Create a new test file in `foo/{good,bad}/bar.catala_<language>` (pick the right directory and
    an informative name for your test)
 2. Write your test, and pick a toplevel scope `A` to run.
 3. Add the following section to your file:
       ~~~markdown
-      ```catala-test {id="A.Interpret"}
+      ```catala-test-inline
       catala Interpret -s A
       ```
       ~~~
-4. Run `CLERK_OPTS=--reset make tests/foo/{good,bad}/bar.catala_<language>`
-   from the root of the Catala repository. This
-   will record the content of the output of your test into the target file.
-5. Check that the expected output is present in `tests/foo/{good,bad}/output/bar.catala_<language>.A.Interpret`
-6. Don't forget to `git add` both the test file and the output file.
+4. Run `make tests/foo/{good,bad}/bar.catala_<language> CLERK_OPTS=--reset`
+   from the root of the Catala repository. This will update the test with the
+   actual output of the catala command.
```

Comment:
Did you add that reset feature also ? It's VERY useful and I don't want us to lose it.

Reply:
yup of course! :)
To ba fair I most often do it this way and then use my git tools to review the diffs